### ISSUE TITLE
Doesn't work without pulseaudio on Pop OS 22.04

### ISSUE THREAD

**Issue by eitch** [rustdesk/rustdesk#559]
The debian package does not install on PopOS 22.04 due to it not using pulseaudio.

...

It would be awesome if you could support the alternative too.

...

Debian package from the website.

---

**Comment by rustdesk** [comment]
Can you install pulseaudio?

**Comment by dtantono** [comment]
Well, I have to admit pipewire is the future since Ubuntu want to migrate to pipewire and many distros like it and I hope rustdesk can do that as well.

**Comment by rustdesk** [comment]
Noted, Thanks

**Comment by eitch** [comment]
@rustdesk Pulseaudio can't be installed as well:

    $ sudo apt install pulseaudio
    Please touch the device.
    Reading package lists... Done
    Building dependency tree... Done
    Reading state information... Done
    Some packages could not be installed. This may mean that you have
    requested an impossible situation or if you are using the unstable
    distribution that some required packages have not yet been created
    or been moved out of Incoming.
    The following information may help to resolve the situation:

    The following packages have unmet dependencies:
     pop-desktop : Conflicts: pulseaudio
                   Recommends: io.elementary.sideload but it is not installable
                   Conflicts: pulseaudio:i386
    E: Error, pkgProblemResolver::Resolve generated breaks, this may be caused by held packages.

**Comment by gerdesj** [comment]
Pipewire has a Pulse compatibility layer which is what I use on Arch.  See if you have a package called pipewire-pulse or similar.

**Comment by eitch** [comment]
This would probably be the solution, but the debian package needs to reference pipewire-pulse as an alternative, and not have a hard dependency on pulseaudio

**Comment by rustdesk** [comment]
https://github.com/rustdesk/doc.rustdesk.com/issues/41#issuecomment-1163882048

**Comment by Gcat101** [comment]
Still waiting

**Comment by ggobbe** [comment]
Or without repackaging you can force the installation with:

```
sudo dpkg --force-all -i rustdesk-1.1.9.deb
```

I also had installed `pipewire-pulse`.

**Comment by ZeddieXX** [comment]
Great workaround.  But when will this be fixed in the official download?  Also when will devs submit this to default repos of major distros (Ubuntu, PopOS, Arch, Fedora, etc)?

**Comment by koutheir** [comment]
This is also became an issue on Debian Testing as of today, as pipewire is replacing pulseaudio, and removing rustdesk due to unsatisfied requirements.

**Comment by rustdesk** [comment]
try out nightly build, pulseaudio dep removed

**Comment by rmiddle** [comment]
> ok, so repackaging allows me to install and then run rustdesk:
> 
> ```
> $ mkdir -p newpack oldpack/DEBIAN
> $ dpkg-deb -x rustdesk-1.1.9.deb oldpack
> $ dpkg-deb -e rustdesk-1.1.9.deb oldpack/DEBIAN/
> 
> # modify control
> $ sed -i "s/pulseaudio/pulseaudio | pipewire-pulse/g" oldpack/DEBIAN/control
> 
> $ dpkg-deb -Z xz -b oldpack newpack
> sudo dpkg -i newpack/rustdesk_1.1.9_amd64.deb
> ```

This worked for me and another guy I know.  If you want to use the current release, this steps work and don't actually change anything other than the depends on the installer.

**Comment by TailsTyler** [comment]
So how can i avoid the "Child process exited with code 100" error in layman's terms? Just copy these into terminal? https://github.com/rustdesk/rustdesk/issues/559#issuecomment-1312749660

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/issues/3203
